github/src/main/kotlin/gropius/sync/github/model/RepositoryInfo.kt

```diff
@@ -8,6 +8,7 @@ import java.time.OffsetDateTime
 
 /**
  * Mapping of a single repository from neo4j to github
+ * @param imsProject IMSProject of the repo
```

Comment:
please be consistent with the way you document params in one class

Reply:
af217fde871ded5eb3e03a21b7d408b96b04189a